opendevin/state.py

```diff
@@ -0,0 +1,6 @@
+from dataclasses import dataclass
+from typing import Mapping
+
+@dataclass
+class State:
```

Comment:
I wonder if we should keep the Event/Action history here too

Reply:
Good catch! will add